api/src/wfl/module/covid.clj

```diff
@@ -708,69 +820,105 @@
   (when-let [snapshot (peek-queue! source)]
     (let [reference  (from-source executor snapshot)
           submission (create-submission! executor reference)]
-      (write-workflows! executor reference submission)
+      (allocate-submission executor reference submission)
       (pop-queue! source)))
+  (update-unassigned-workflow-uuids! executor)
   (update-terra-workflow-statuses! executor)
   executor)
 
+(defn ^:private from-firecloud-workflow [methodConfig workflow]
```

Comment:
Just want to make sure the aliased `methodConfig` here is intentional

Reply:
yep -i was lazy - the others are form map de-structuring.